protocol/v2/ssv/validator/committee.go

```diff
@@ -98,6 +100,7 @@ func (c *Committee) StartDuty(logger *zap.Logger, duty *spectypes.CommitteeDuty)
 		return errors.New(fmt.Sprintf("CommitteeRunner for slot %d already exists", duty.Slot))
 	}
 
+	attestingValidators := make([]spectypes.ShareValidatorPK, 0, len(duty.BeaconDuties))
```

Comment:
```suggestion
	slashableValidators := make([]spectypes.ShareValidatorPK, 0, len(duty.BeaconDuties))
```
I think this name is better